sdk/sdk-skeleton.go

```diff
@@ -0,0 +1,77 @@
+type Asset struct{}
+
+//// Methods directly on the SDK
+
+func NewChannel() {
+
+}
+
+func NewChannelListenerService() {
+
+}
```

Comment:
This can be removed since there is no listener service yet.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/25/commits/71a7168df8ac07dc6ac331728a5910c64328add3